src/hooks/cluster/useStorageClasses.ts

```diff
@@ -0,0 +1,40 @@
+import { useSetAdminClusterSidebar } from "@/utils/helpers";
+import moment from "moment";
+
+const useStorageClasses = ({ cluster_id }: any) => {
+  useSetAdminClusterSidebar();
+
+  const tableColumns = () => [
+    { id: "name", header: "Name" },
+    { id: "provisioner", header: "Provisioner" },
+    { id: "reclaimPolicy", header: "Reclaim Policy" },
+    //{ id: "default", header: "Default" },
+    { id: "age", header: "Age" },
+  ];
+
+  const tableData = (data: any) => {
+    if (!data) {
+      return [];
+    }
+    if (!Array.isArray(data)) {
+      return [];
+    }
+    return data.map((item: any) => {
+      const row = {
+        // ...item,
+        name: item?.metadata?.name,
+        provisioner: item?.provisioner,
+        reclaimPolicy: item?.reclaimPolicy,
+        //default: formatClusterServicePorts(item?.spec?.default),
```

Comment:
You need to add default storage class, it is very important.  
I think you can get it from:
`item>metadata>annotations>`, then look for one that has this key set to true `storageclass.kubernetes.io/is-default-class`